src/lib/select/select-animations.ts

```diff
@@ -73,6 +74,18 @@ export const matSelectAnimations: {
       style({opacity: 0}),
       animate('150ms 100ms cubic-bezier(0.55, 0, 0.55, 0.2)')
     ])
+  ]),
+
+  /**
+   * This animation moves the panel toggle arrow up whenever the select is not empty to match the
+   * visual middle. This affects only mat-selects with appearance='standard'.
+   */
```

Comment:
Defining an animation just to move the arrow around is overkill and it makes it very hard for people to override the styles. Since the `empty` state is on the `mat-select` already, you can use it to toggle a class either on the select or the arrow itself which tells it where it's supposed to be positioned.

Reply:
Using a normal CSS transition will make it easy to target the specific appearance as well, since the form field already has a class indicating the variant.

You can hook into the Angular Animations to disable this transition when Angular Animations are disabled. As an example see here: https://github.com/angular/material2/blob/master/src/lib/form-field/form-field.scss#L220

Reply:
Yeah, using just simple CSS seems way better and easier to handle - I'm gonna update the PR on the weekend!